test/modules/core/lib/deck.spec.ts

```diff
@@ -61,7 +61,7 @@ test('Deck#no views', t => {
     device,
     width: 1,
     height: 1,
-    autoResizeDrawingBuffer: device.canvasContext.htmlCanvas.clientWidth > 0,
+    autoResizeDrawingBuffer: device.canvasContext.canvas.clientWidth > 0,
```

Comment:
```suggestion
    autoResizeDrawingBuffer: device.getCanvasContext().canvas.clientWidth > 0,
```

Reply:
Nit: by accessing clientWidth directly, this becomes incompatible with `OffscreenCanvas`. `CanvasContext` does provide methods that help apps encapsulate such things, though I am the first to admit that the CanvasContext class does need more docs and tests. 

That can of course easily be improved in a future PR should it ever be needed.